.gitignore

```diff
@@ -19,5 +19,5 @@ examples/*/ref
 examples/*/test
 *.swp
 check_isa.exe
-
+build
```

Comment:
What about `build*`?

Reply:
Wasn't sure if there would ever be a script in the root starting with `build`. It's probably safe to do `build*`.

Reply:
If we would add such script, we'll remove the `.gitignore`. Though I routinely do have more than build folder in the workspace.

Reply:
```suggestion
build*
```